.circleci/config.yml

```diff
@@ -345,20 +345,20 @@ workflows:
   build_and_package:
     jobs:
       - lint:
-          <<: *on-any-branch
+          <<: *not-on-integ-branch
```

Comment:
@rafie why cancel it on integration branches?

Reply:
@danni-m We'll always have this on PR (non-integ) branches, so it's basically an annoyance (preventing one from merging a PR) or a mere duplication. If by an obscure way someone has managed to bypass lint on a PR, the next PR will catch it. WDYT?

Reply:
@rafie OK, i see.